libs/output-mapping/src/Writer/Table/TableDefinition/TableDefinitionColumnFactory.php

```diff
@@ -5,13 +5,22 @@
 namespace Keboola\OutputMapping\Writer\Table\TableDefinition;
 
 use Keboola\Datatype\Definition\Common;
+use Keboola\Datatype\Definition\DefinitionInterface;
 
 class TableDefinitionColumnFactory
 {
+    private ?string $nativeDatatypeClass;
+
+    public function __construct(?string $nativeDataTypeClass)
```

Comment:
Took me some time to figure out that this is not really class, but class name, so the type hint should be `class-string<DefinitionInterface> https://phpstan.org/writing-php-code/phpdoc-types#class-string